assets/sass/v2/_device-challenge.scss

```diff
@@ -47,6 +47,9 @@
     .highlight-text {
       font-weight: bold;
     }
+    .app-link-content {
```

Comment:
do you think this should be something generically handled by baseForm subtitles ?